assets/scss/optimized-checkout.scss

```diff
@@ -0,0 +1,180 @@
+// =============================================================================
+// Stencil Checkout - Customize the Optimized Single-Page Checkout experience
+// =============================================================================
+
+// =============================================================================
+//
+// IMPORTANT DISCLAIMER
+// Please refrain from nesting or using other class selectors than the ones given below.
+// Future support is only guaranteed if class selectors are left unchanged.
+//
+// =============================================================================
+
+//
+// Import Mixins/Tools
+// breakpoint
+// -----------------------------------------------------------------------------
+
+@import "../../node_modules/@bigcommerce/citadel/dist/tools/toolkit"; // 1
+@import "settings/global/global"; // 1
+@import "../../node_modules/@bigcommerce/citadel/dist/settings/global/global"; // 2
+
+//
+// Header Section
+// Background, Logo Position
+// -----------------------------------------------------------------------------
+
+.optimizedCheckout-header {
+    background-color: stencilColor("optimizedCheckout-header-backgroundColor");
+    // scss-lint:disable StringQuotes
+    background-image: url(stencilImage('optimizedCheckout-backgroundImage', 'optimizedCheckout-backgroundImage-size'));
+    // scss-lint:enable StringQuotes
+    background-size: cover;
+
```

Comment:
I would prefer to keep `background-size: cover;` since we're using the same image for all responsive sizes and this guarantees that there won't be any blank spaces at the edges of the header.

As for if we should be allowing for different image sizes - I think having one size makes more sense. If we allow them to upload smaller image sizes, we run the risk that the images will be stretched in different screen sizes. Currently in the code, we don't do anything special with the design at different breakpoints for us to allow for variable sizes.